openrtb/bidresponse.go

```diff
@@ -71,12 +71,14 @@ func (r *BidResponse) Copy() *BidResponse {
 
 	brCopy := *r
 
-	brCopy.SeatBids = []*SeatBid{}
-	for _, seat := range r.SeatBids {
-		brCopy.SeatBids = append(brCopy.SeatBids, seat.Copy())
+	if r.SeatBids != nil {
+		brCopy.SeatBids = make([]*SeatBid, len(r.SeatBids))
+		for i, seat := range r.SeatBids {
+			brCopy.SeatBids[i] = seat.Copy()
+		}
 	}
 
-	if r.Extension != nil {
+	if r.RawExtension != nil {
```

Comment:
It seems the modification is OK. I suggest to confirm the change with @xiaopeng0216

Reply:
Revert this change to ensure jaeger will not be broken. This strange logic should be fixed in future.